### ISSUE TITLE
[docs] Swagger docs errored in production

### ISSUE THREAD

**Issue by atb00ker** [openwisp/openwisp-radius#192]
I am getting on a production server:
```
drf_yasg.errors.SwaggerGenerationError: Schema for <class 'openwisp_users.api.swagger.ObtainTokenRequest'> 
would override distinct serializer <class 'openwisp_radius.api.swagger.ObtainTokenRequest'> because they 
implicitly share the same ref_name; explicitly set the ref_name atribute on both serializers' Meta classes
```

**Comment by nemesisdesign** [comment]
@atb00ker does this error show up on startup or when accessing a specific URL or when exactly? I didn't notice this yet.

**Comment by atb00ker** [comment]
@nemesisdesign shows up when you access the the `/api/v1/docs` and check the logs in server at `/opt/openwisp/logs`. :smile:

**Comment by nemesisdesign** [comment]
Fixed by 34f8df486980c513e0403e3b085a170f1ae5ce4e
